pkg/cloudevents/event.go

```diff
@@ -13,12 +13,26 @@ type Event struct {
 	Data        interface{}
 	DataEncoded bool
 	DataBinary  bool
+	FieldErrors map[string]error
```

Comment:
users will check whether `event.FieldErrors != nil`, right?

Reply:
discussed offline

Reply:
so no need to call Validate on caller code then... that is done in the SDK in client.Send

Reply:
sycned offline